pkg/storage/driver/secrets_test.go

```diff
@@ -245,7 +246,7 @@ func TestSecretDelete(t *testing.T) {
 
 	// fetch the deleted release
 	_, err = secrets.Get(key)
-	if !reflect.DeepEqual(ErrReleaseNotFound, err) {
```

Comment:
I guess strictly, `reflect.DeepEqual` does more than `errors.Is`. But should be equivalent in this case

Reply:
Lmk if I should revert